config/boot.rb

```diff
@@ -1,4 +1,15 @@
 # Set up gems listed in the Gemfile.
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
 
-require 'bundler/setup' if File.exist?(ENV['BUNDLE_GEMFILE'])
+require 'bundler/setup' # Set up gems listed in the Gemfile.
+
+# load dotenv files before "before_configuration" callback
+require File.expand_path('../configuration', __FILE__)
+
+# global instance to access and use
+AppConfig = Configuration.new
```

Comment:
The `AppConfig` variable doesn't follow typical Ruby naming conventions:

https://github.com/bbatsov/ruby-style-guide#naming

They recommend camel case for `Class` or `Module` only.

Reply:
addressed out offline.